app/views/spree/admin/users/_form.html.haml

```diff
@@ -23,4 +23,11 @@
     = f.field_container :password do
       = f.label :password_confirmation, t(".confirm_password")
       = f.password_field :password_confirmation, class: "fullwidth"
-      = f.error_message_on :password_confirmation
\ No newline at end of file
+      = f.error_message_on :password_confirmation
+    = f.field_container :disable do
+      = f.label :disable, t(".disable")
+      =# check_box_tag "user[toggle_disable]", "1", @user.disabled?, id: "user_toggle_disable"
+      = f.check_box :toggle_disable
+      = f.error_message_on :disable
+      - if @user.disabled?
+        = label_tag @user.disabled_at
```

Comment:
Displaying the time was not in the requirements. Let's keep the interface simple.